community/dogefetch/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+make install
```

Comment:
I didn't notice this earlier but the `Makefile` for `dogefetch` defaults to `/usr/local` (https://github.com/ali019283/dogefetch/blob/ded7bbed548dfb8e47488fbd59149f52a444c2ef/Makefile#L3), and this should be installed to `/usr/bin`.